### PR TITLE
Address flakiness of LinuxTracingIntegrationTest

### PR DESCRIPTION
**Pull request by dpallotti**
Bug: http://b/181634108

### CODE REVIEW

File: src/LinuxTracing/UprobesUnwindingVisitor.cpp [hunk 1]
```diff
@@ -44,13 +44,24 @@ void UprobesUnwindingVisitor::visit(StackSamplePerfEvent* event) {
...
 
+  // LibunwindstackUnwinder::Unwind signals an unwinding error with an empty callstack.
   if (libunwindstack_callstack.empty()) {
     if (unwind_error_counter_ != nullptr) {
       ++(*unwind_error_counter_);
     }
     return;
   }
 
+  // Callstacks with only one frame (the sampled address) are also unwinding errors, that were not
```
Comment (florian-kuebler) [inline src/LinuxTracing/UprobesUnwindingVisitor.cpp:55]: What is if we have a sample inside main?

Comment (dpallotti) [reply src/LinuxTracing/UprobesUnwindingVisitor.cpp:55]: This is not the case of main because there are other frames outside of main.

Comment (florian-kuebler) [reply src/LinuxTracing/UprobesUnwindingVisitor.cpp:55]: I see. Could you add that concrete statement to the comment? Otherwise, LGTM.

**Review by florian-kuebler — APPROVED** [review]

File: src/LinuxTracing/UprobesUnwindingVisitor.cpp (hunk 1 above)
Comment (dpallotti) [reply src/LinuxTracing/UprobesUnwindingVisitor.cpp:55]: Good point, done.

**Pull request merged**